azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -281,5 +365,46 @@ func getArmTrafficManagerEndpointProperties(d *schema.ResourceData) *trafficmana
 		endpointProps.MinChildEndpoints = &mci64
 	}
 
+	templist := d.Get("subnet").([]interface{})
+	subnetMappings := make([]trafficmanager.EndpointPropertiesSubnetsItem, 0)
+	for _, subnetOld := range templist {
```

Comment:
We could simplify this:
```suggestion
	for _, subnetOld := range d.Get("subnet").([]interface{}) {
```